src/Wallabag/CoreBundle/Resources/views/themes/baggy/Entry/entry.html.twig

```diff
@@ -29,6 +29,11 @@
             <li><a href="mailto:hello@wallabag.org?subject=Wrong%20display%20in%20wallabag&amp;body={{ entry.url|url_encode }}" title="{{ 'entry.view.left_menu.problem.label'|trans }}" class="tool bad-display icon icon-delete"><span>{{ 'entry.view.left_menu.problem.label'|trans }}</span></a></li>
         </ul>
     </div>
+        <div class="link mdi-action-today"> {{ entry.createdAt|date('Y-m-d') }}</div>
+        {% set readingTime = entry.readingTime / app.user.config.readingSpeed %}
+        <div class="link mdi-action-query-builder"> {% if readingTime > 0 %}{{ 'entry.list.reading_time_minutes_short'|trans({'%readingTime%': readingTime|round}) }}
+            {% else %}{{ 'entry.list.reading_time_less_one_minute_short'|trans|raw }}
+            {% endif %}</div>
```

Comment:
Can't this be displayed more nicely? I mean, it's hard to read the if / else / endif with that kind of indentation.